src/app/static/src/app/main.ts

```diff
@@ -14,7 +14,7 @@ export interface RootState {
 }
 
 export interface StepState {
-    complete: boolean
+    complete: () => boolean
```

Comment:
I wondered if this would work better as a function, since it's likely to be a value derived from other properties in the state.

Reply:
Will this work? I don't think the function will be automatically executed when the underlying data changes. This could/should be tested by instantiating the component with incomplete data, then triggering the action/mutation which makes the stage complete, then checking whether the continue button is enabled.

It looks like vue does have a concept of "getters" aka computed properties for the store which I think are what you're after here: https://vuex.vuejs.org/guide/getters.html